pavelib/utils/test/utils.py

```diff
@@ -150,3 +157,35 @@ def check_firefox_version():
                 debian_path=debian_path
             )
         )
+
+
+@task
+@cmdopts([
+    ("compare-branch=", "b", "Branch to compare against, defaults to origin/master"),
+], share_with=['coverage'])
```

Comment:
I might be confused about what share_with means. I thought this meant that this task would @need coverage, and this option would be shared with it.

Reply:
I think I fixed it?